pmd-gherkin/pom.xml

```diff
@@ -18,6 +18,28 @@
                 <artifactId>antlr4-maven-plugin</artifactId>
             </plugin>
 
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>antlr-cleanup</id>
```

Comment:
Maybe we should extend the documentation as well here: https://github.com/pmd/pmd/blob/bc4a49deacdf8100097a31af5aa22477a19a16ed/docs/pages/pmd/devdocs/major_contributions/adding_new_cpd_language.md?plain=1#L23-L27

For both Antlr-based and Javacc-based CPD implementations, we need to run some ant script.

Reply:
Yes, that's in my backlog, along the other 3-4 languages still pending these fixes